### DISCUSSION TITLE
Specify which folder to sync to (Folder on Onedrive)

### DISCUSSION THREAD

**Discussion by realrEDGUY** [abraunegg/onedrive#1358] [Q&A] [answered]
Hey, I was wondering if it is possible to sync to a specific folder.

So if I for example had 2 servers, then I'd want Server X to upload to Folder-X.

Instead of having a pile of everything together

**Comment by abraunegg** [comment, marked as answer]
@realrEDGUY 
Please read the usage document in regards to:
* Single Directory
* Upload Only

**Reply by realrEDGUY** [reply]
Not  exactly what i was looking for, it still uploads as "backups" instead of "Server-1/Backups"

**Reply by abraunegg** [reply]
OneDrive Folder 'redirection' is *not* possible because the OneDrive API does not support this.

Your options are:

* Store all your data on Server-1 in Folder X and use `--single-directory` to upload to the required folder
* Use separate OneDrive accounts

**Reply by realrEDGUY** [reply]
Plesk makes it possible, they put it into the "Apps" folder as "Apps/plesk-backups"

But I can't choose which folder specifically that's why I using this

**Reply by abraunegg** [reply]
OneDrive Folder 'redirection' is not possible because the OneDrive API does not support this.

What Plesk is doing is forcing essentially a 'single directory' which is a modified / mangled directory root, which is exactly the same solution articulated above:

* Store all your data on Server-1 in Folder X and use `--single-directory` to upload to the required folder

**Example:**
sync_dir='~/OneDrive'

Server X data stored in local folder 'this-servers-data' - which is a sub folder of '~/OneDrive'

**Command:** `onedrive --synchronize --single-directory 'this-servers-data'`
